kindcat_custom.cue

```diff
@@ -9,7 +9,9 @@ import (
 // _kubeObjectMetadata is metadata found in a kubernetes object's metadata field.
 // It is not exhaustive and only includes fields which may be relevant to a kind's implementation,
 // As it is also intended to be generic enough to function with any API Server.
-_kubeObjectMetadata: {
+KubeObjectMetadata: {
+	namespace: string
+	name: string
```

Comment:
it feels odd to qualify these as kube metadata... since they are essentially the identifiers.  I was expecting kube metadata to be the more essoteric k8s specific things like:
* finalizers
* managedFields
* deletionGracePeriodSeconds
* etc

Reply:
This is mainly to make them part of what kubernetes considers "metadata," i.e. what kubernetes puts in its `metadata` field, so that `KubeMetadata` is exactly compatible with the `metadata` field in kubernetes.

We _could_ present a different interface via `Resource`, but I think we'd want generated code/`UnstructuredResource` to be formatted the same way the kubernetes object itself is.